### ISSUE TITLE
Uninstalling HKCantonIME

### ISSUE THREAD

**Issue by mangoose88** [556556bt/HKCantonIME#30]
Does anyone know how to uninstall this in Win10? This keeps crashing in Word when I press "z." I don't see it in the settings/preferred languages section and you can't right click to uninstall in the lower right tray menu

**Comment by Kaionman** [comment]
You can uninstall from the control panel, while you uninstalled it, there will be an Traditional Chinese keyboard layout in english remain. This is very bothering.

**Comment by Khanagec** [comment]
> You can uninstall from the control panel, while you uninstalled it, there will be an Traditional Chinese keyboard layout in english remain. This is very bothering.

Yeah I've been trying to remove that layout for the last couple days from registry. I've managed to get rid of it temporarily but it came back today.

**Comment by badbeing** [comment]
Hello guys I think I might have found the solution (although many years late...)
I've had similar case when I tried to uninstall CPIME on Windows 11, which left residue files in the system causing a Trad. Chinese keyboard layout shows up in language list permanently. After digging I found a PowerShell script that is used to clean up all Chinese input methods and install CPIME alternatively.

The link of the article that contains the PowerShell script (**For the "2004" update** Section):
[https://www.pinyinjoe.com/faq/windows-10-cantonese-phonetic-ime-cpime.htm](https://www.pinyinjoe.com/faq/windows-10-cantonese-phonetic-ime-cpime.htm)

And for people who just want to solve the problem fast, the direct link of the script inside a zipped file:
[https://www.pinyinjoe.com/downloads/CPIME/2004%20fix/cpime-2004.zip](https://www.pinyinjoe.com/downloads/CPIME/2004%20fix/cpime-2004.zip)

Reminder that you **need** to modify the script so that it **only** serves as a cleanup tool (# Comment out Line 7 as follow)
**#**`$hanthk.InputMethodTips.Add('0C04:E03D0C04')`
Then **Save** the file and right click to **Run with PowerShell**
